configs/cliff/resnet50_pw3d_cache.py

```diff
@@ -0,0 +1,225 @@
+_base_ = ['../_base_/default_runtime.py']
```

Comment:
Please resolve the following:
1. Resolve the merge conflicts with the current main branch
2. Add a README in configs/cliff/, [example](https://github.com/open-mmlab/mmhuman3d/blob/main/configs/pare/README.md)
3. Add tests, [example](https://github.com/open-mmlab/mmhuman3d/commit/46d3dae60c5018c8940102f8ddfbe76661f250ce)